pkg/monitor/api.go

```diff
@@ -75,22 +43,22 @@ func Start(ctx context.Context, restConfig *rest.Config) (*Monitor, error) {
 		return nil, err
 	}
 
-	if err := StartKubeAPIMonitoringWithNewConnections(ctx, m, restConfig, 5*time.Second); err != nil {
+	if err := startKubeAPIMonitoringWithNewConnections(ctx, m, restConfig); err != nil {
```

Comment:
This call is what prevents us from moving the disruptionmonitor logic out of this package.  We need a way to inject additional monitors from outside the package so we can late-bind them to avoid the cycle.  The risk is then drift among the various tests, but I think it's tolerable.  followup.